kratos/includes/model_part.h

```diff
@@ -843,6 +843,65 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
      */
     PropertiesType& GetProperties(IndexType PropertiesId, IndexType MeshIndex = 0) const;
 
+    /**
+     * @brief Returns if the sub Properties corresponding to it's address exists
+     * @param rAddress The text that indicates the structure of subproperties to iterate and found the property of interest
+     * @param ThisIndex The index identifying the mesh
+     * @return True if the properties exist, false otherwise
+     */
+    bool HasSubProperties(
+        const std::string& rAddress,
+        IndexType MeshIndex = 0
+        ) const;
+
+    /**
+     * @brief Returns the sub Properties::Pointer  corresponding to it's address
+     * @details If the property is not existing it will return a warning
+     * @param rAddress The text that indicates the structure of subproperties to iterate and found the property of interest
+     * @param MeshIndex The Id of the mesh (0 by default)
+     * @return The desired properties (pointer)
+     */
+    PropertiesType::Pointer pGetSubProperties(
```

Comment:
i would propose removing this method (once again @loumalouomega , do not do anything, until we find an agreement)